src/Transition/InputRulesFiles/Rules22-2-0-to-23-1-0.md

```diff
@@ -9,263 +9,3 @@ This was previously an Excel workbook that made for very difficult version contr
 Describe input change here
 
 Use links like: [PR#9480](https://github.com/NREL/EnergyPlus/pull/9480/)
-
-# Object Change: Coil:Cooling:DX:TwoSpeed
```

Comment:
These changes should have gone in the 23.1 to 23.2 file, so I put it in over there.